README.md

```diff
@@ -83,7 +89,27 @@ See [customize options] and [manage connection profiles] for more details.
 ```
 
 ## Change Log
-The current version is ```0.2.0```. See the [change log] for more detail.
+The current version is ```0.2.0```. See the [change log] for a detailed list of changes in each version.
+
+## Supported Operating Systems
+
+Currently this extension supports the following operatings systems:
+
+* Windows (64-bit only)
+* macOS
```

Comment:
We build for 10.11 (El Capitan) but run on 10.10 and 12 (Sierra). Effectively should work in all cases